proctor-store-git/build.gradle

```diff
@@ -19,4 +23,44 @@ dependencies {
     testImplementation libs.assertj
 }
 
-test.dependsOn(':proctor-common:shadowJar')
\ No newline at end of file
+test.dependsOn(':proctor-common:shadowJar')
+test.dependsOn(':proctor-store:shadowJar')
+
+shadowJar.dependsOn(':proctor-store:shadowJar')
+shadowJar.dependsOn(':proctor-common:shadowJar')
+
+// Need to remove all dependenceis that get added or will cause issues upstream when projects include
+// Plan to remove in PROC-1290: Remove Proctor Shading of Javax El libraries
+shadowJar {
+    archiveClassifier.set('')
+    dependencies {
+        exclude(dependency('org.springframework:spring-core:5.3.19'))
```

Comment:
Do we need to maintain the version of the dependency that it excludes?

Reply:
Removed in all excludes